### PR TITLE
Update example scenario in README.md

### PR DESCRIPTION
**Pull request by pmacik**
This PR updates the Service Binding Demo example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,17 +79,57 @@ EOS
 
 ## Example Scenario
 
-### What the Cluster Admin does
+In this example there a are 2 roles:
```
Comment (Avni-Sharma) [inline README.md]: nit pick `there are`

File: README.md [hunk 2]
```diff
@@ -79,17 +79,57 @@ EOS
...
+### Cluster Admin
 
-``` yaml
+The cluster admin needs to install 2 operators into the cluster:
+
+* Service Binding Operator
+* Backing Service Operator
+
+A Backing Service Operator that is "bind-able," in other
+words a Backing Service Operator that exposes binding information in secrets, config maps,status, and/or spec
```
Comment (ldimaggi) [inline README.md]: Very small nit - need a space after "config maps"

File: README.md [hunk 3]
```diff
@@ -79,17 +79,57 @@ EOS
...
+attributes. The Backing Service Operator may represent a database or other services required by
+applications. We'll use [postgresql-operator](https://github.com/baijum/postgresql-operator) to
+demonstrate a sample use case.
+
+#### Install the Service Binding Operator using and `OperatorSource`:
```
Comment (ldimaggi) [inline README.md]: should be:   using **an** 'OperatorSource'

File: README.md [hunk 4]
```diff
@@ -100,76 +140,122 @@ spec:
...
+* `Application`->`Create New Application` = `NodeJSApp`
+* `Name` = `nodejs-app`
+* `Builder Image` = `Node.js`
+* `Create a route to the application` = checked
+
+and click on the `[Create]` button.
+
+Notice, that during the import no DB config was mentioned or requestd.
+
+When the application is running navigate to it's route to verify that it is up. Notice that in the header it says `(DB: N/A)`. That meens that the application is not connected to a DB and so it should not work properly. Try the application's UI to add a fruit - it causes an error proving that the DB is not connected.
```
Comment (ldimaggi) [inline README.md]: Small grammar nit:  Should be "its" not "it's"

File: README.md [hunk 5]
```diff
@@ -100,76 +140,122 @@ spec:
...
+* `connects-to=postgres` - indicates that the application needs to connect to a PostgreSQL DB
+* `environment=demo` - indicates the demo environment - it narrows the search
+
+
+``` bash
+ kubectl patch dc nodejs-app -p '{"metadata": {"labels": {"connects-to": "postgres", "environment":"demo"}}}'
 ```
 
-### What the Application Author (Developer) does
+#### Expres an intent to bind the DB and the application
```
Comment (ldimaggi) [inline README.md]: Small nit: Should be: Express

File: README.md (hunk 1 above)
Comment (pmacik) [reply README.md]: Thanks @Avni-Sharma  :) Fixed

File: README.md (hunk 2 above)
Comment (pmacik) [reply README.md]: Thanks @ldimaggi I like your precision-attitude ;)
Fixed

File: README.md (hunk 4 above)
Comment (pmacik) [reply README.md]: Thanks @ldimaggi, Fixed

File: README.md (hunk 3 above)
Comment (pmacik) [reply README.md]: Thanks @ldimaggi, Fixed

File: README.md (hunk 5 above)
Comment (pmacik) [reply README.md]: Thanks @ldimaggi, Fixed

**Pull request merged**